openapi/components/schemas/Common/CommonKhelocardCard.yaml

```
@@ -1,5 +1,6 @@
 type: object
 title: 'Khelocard Card'
+x-abstract-parent: PaymentInstrument
 properties:
   id:
     description: The payment instrument ID.
```

nickpick: suggestion to avoid starting the description with "the"
```suggestion
    description: ID of the payment instrument.
```